packages/core/src/node/uploadContribution.ts

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
please follow naming conventions for files: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#names
> Use lower case, dash-separated file names (e.g. document-provider.ts)

Reply:
Ok, rename file from "uploadContribution.ts"  to "upload-contribution.ts"